.github/contributing/integrations/MEMORY.md

```diff
@@ -0,0 +1,22 @@
+# Contributing third-party memory
+
+This page contains some specific guidelines and examples for contributing integrations with third-party memory providers.
```

Comment:
Let's cross link here to MESSAGE_STORES, as many times when people think they want to add a memory class they actually want to add a message store